### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by timocov**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Issues
+
+### Search for duplicates
+
+[Search the existing issues](https://github.com/tradingview/lightweight-charts/search?type=Issues) before logging a new one.
+
+*Don't* restrict your search to only open issues. An issue with a title similar to yours may have been closed as a duplicate of one with a less-findable title.
+
+### Logging a bug
```
Comment (subzey) [inline CONTRIBUTING.md]: "Filing an Issue" is a more widespread term for doing this (If I understand it correctly)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+When logging a bug, please be sure to include the following:
+
+- What version of the library you're using.
+- If at all possible, an *isolated way* to reproduce the behavior.
+- The behavior you expect to see, and the actual behavior (e.g. screenshots).
+
```
Comment (subzey) [inline CONTRIBUTING.md]: Make sure the issue title is understandable and searcheable.

*Bad:* Oh my god! This goddamn thing just blew up! Is that my torn off finger laying out there? Oh my freaking god!

*Good:* ACME graphics card overheats and causes injuries when using the software

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Pull requests
+
+For build instructions/suggestions see [BUILDING.md](./BUILDING.md).
+
+### Tests
+
+1. Every pull request should have an adequate tests whenever it's possible (we have several [type of tests](./tests/), so you can find what better fit for your changes).
```
Comment (ezhukovskiy) [inline CONTRIBUTING.md]: so you can find what works best for your changes

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+1. If you change something affects painting, your changes should have a test case(s) for [graphics tests](./tests/e2e/graphics).
```
Comment (ezhukovskiy) [inline CONTRIBUTING.md]: If your changes affect paining, then your changes should contain a test case (or test cases) for [graphics tests](./tests/e2e/graphics).

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+1. Your pull request should have passed CI (except checks marked as "not required" - in this case reviewer should pay attention on job's artifacts).
```
Comment (ezhukovskiy) [inline CONTRIBUTING.md]: Your pull request should pass CI (except checks marked as "not required" - in this case a reviewer should pay attention to job's artifacts).

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+### Git commit messages
+
+1. Please make sure that every of your commits have clear commit message (not just `fix bug` or something - describe a little what's changed/fixed).
```
Comment (ezhukovskiy) [inline CONTRIBUTING.md]: Please make sure that every your commit has a clear commit message (not just `fix bug` or something like this - describe what has been changed/fixed).

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+1. If your commit addressed to GitHub issue (or even for a comment), feel free to add it somehow:
```
Comment (ezhukovskiy) [inline CONTRIBUTING.md]: If your commit addresses a GitHub issue, feel free to add it somehow:

**Pull request merged**